xrmocap/data/dataset/fourd_dataset.py

```diff
@@ -0,0 +1,386 @@
+import glob
+import logging
+import numpy as np
+import os
+import re
+import torch
+import pickle
+import cv2
+from typing import Tuple, Union
+from xrprimer.data_structure.camera import FisheyeCameraParameter
+
+from xrmocap.data_structure.keypoints import Keypoints
+from xrmocap.transform.convention.keypoints_convention import convert_keypoints
+from .base_dataset import BaseDataset
+
+try:
+    from typing import Literal
+except ImportError:
+    from typing_extensions import Literal
+
+
+class FourDDataset(BaseDataset):
```

Comment:
What about `class FourDDataset(MviewMpersonDataset)`? As I've found some similar functions in FourDDataset.

Reply:
I will adopt it